std/http/cookie.ts

```diff
@@ -114,6 +115,31 @@ function validatePath(path: string | null): void {
   }
 }
 
+/**
+ *Validate Cookie Value.
+ * @see https://tools.ietf.org/html/rfc6265#section-4.1
+ * @param value Cookie value.
+ */
+function validateCookieValue(value: string | null): void {
+  if (value == null) return;
+  for (let i = 0; i < value.length; i++) {
+    const c = value.charAt(i);
+    if (
+      c < String.fromCharCode(0x21) || c == String.fromCharCode(0x22) ||
+      c == String.fromCharCode(0x2c) || c == String.fromCharCode(0x3b) ||
+      c == String.fromCharCode(0x5c) || c == String.fromCharCode(0x7f)
+    ) {
+      throw new Error("RFC2616 cookie value cannot have '" + c + "'");
```

Comment:
Nitpick, include cookie name in the error

Reply:
what do you think ? 
 ```js
"RFC2616 cookie '" + name + "' cannot have '" + c + "' as value" 
```